README.md

```diff
@@ -69,7 +69,10 @@ spec:
 ## Metadata Collection
 [Metadata Collection](docs/metadata.md)
 
+## Cerberus Integration
+[Cerberus Integration](docs/cerberus.md)
+
 ## Community
-Key Members(slack_usernames): aakarsh, dblack or rook
+Key Members(slack_usernames): aakarsh, dry923, rsevilla or rook
```

Comment:
ouch this hurts... @dustinblack has to approve this ;)

Reply:
lol